taskcluster/ci/ui-test/kind.yml

```
@@ -64,3 +64,16 @@ jobs:
                 - [automation/taskcluster/androidTest/ui-test.sh, x86-screenshots-tests, app.apk, android-test.apk, '-1']
         treeherder:
             symbol: debug(screenshots-x86)
+    x86-beta:
+        attributes:
+            build-type: beta
+        description: Test Fenix
+        run-on-tasks-for: [github-push]
+        dependencies:
+            signing: signing-beta-firebase
+            signing-android-test: signing-android-test-beta
+        run:
+            commands:
+                - [automation/taskcluster/androidTest/ui-test.sh, x86-beta-tests, app.apk, android-test.apk, '50']
+        treeherder:
+            symbol: debug(ui-test-x86-beta)
```

Nit:

```suggestion
            symbol: beta(ui-test-x86-beta)
```

This way, it will appear alonside the other beta jobs: https://treeherder.mozilla.org/jobs?repo=fenix&revision=d4ef63f8778f4458565452661bfc6e22766bcd5c&selectedTaskRun=F6aaOEelTrGRcttgoKTRXw.0